charts/stardog/templates/statefulset.yaml

```diff
@@ -49,6 +55,7 @@ spec:
                   - {{ include "stardog.fullname" . }}
               topologyKey: "kubernetes.io/hostname"
         {{- end }}
+{{ if .Values.initContainers.enabled }}
```

Comment:
only for our single node case. though i've not tried it on a cluster. :-D

Reply:
Perhaps it would make sense to use the clusterEnabled flag here? If the cluster is disabled we can skip the init container, but I think it should be required if the cluster is enabled.

Reply:
i think you're right. 
as I pushed the latest, I was thinking the same. I'm usually loathe to couple behaviours, but I can't thing of a reason to decouple these. they are separate in the original, because of our requirement to not deploy busy-box for nothing.